.gitignore

```diff
@@ -3,5 +3,8 @@
 /local.conf
 pEpEngineSequoiaBackendProject.vcxproj.user
 x64
+build-win64/Debug
```

Comment:
I see that you ignore output for win64 only, but what about other platforms? win32, x86 and ARM?